myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html

```diff
@@ -34,19 +32,31 @@
                 <div>
                     <p>Creates multiple email lists, one for each section selected</p>
                     <fieldset>
-                        <div>
-                            <label>Select all</label> <input type="checkbox"/>
+                        <div class="clearfix">
+                            <div class="pull-right">
+                                <label for="select-all-sections">Select all</label> <input id="select-all-sections" type="checkbox"/>
+                            </div>
                         </div>
-                        <ul>
+                        <ul class="clearfix unstyled-list">
                             {{#each secondary_lists}}
-                            <li class="secondary_single">
+                            <li class="secondary_single clearfix">
                                 {{#unless list_exists}}
-                                <label for="secondary_single_{{ list_address }}">
-                                    {{ list_address }}@uw.edu
```

Comment:
`{{ list_address }}@uw.edu` I think this should be the section name. Let me know if I'm misunderstanding your intent though.